MIGRATION.md

```diff
@@ -742,3 +747,319 @@ import {
+    <IconButton
+      aria-label="Open Actions"
+      icon={<MoreVerticalIcon size="small" />}
+    />
+  </Menu.Trigger>
+  <Menu.List style={{ maxHeight: '200px' }}>
+    <Menu.SectionTitle>Actions</Menu.SectionTitle>
+    <Menu.Item>First action</Menu.Item>
+    <Menu.Item>Second action</Menu.Item>
+    <Menu.Divider />
+    <Menu.Item as="a" href="https://contentful.com" target="_blank">
+      About Contentful
+    </Menu.Item>
+  </Menu.List>
+</Menu>
+```
+
+**Case with Submenu:**
+Importent things to take into account in the following example:
+- You can't just pass `Dropdown` as an Menu item. 
+- You have to use specific components like `Menu.Submenu` and `Menu.SubmenuTrigger` to add submenu, but other components like `Menu.List`, `Menu.Item`, `Menu.Divider`, etc. remain the same.
+- Submenu will be always shown on the right side. You can't stick it to the left.
+
+Forma v3:
+
+```tsx static=true
+import { Dropdown, DropdownList, DropdownListItem, IconButton } from '@contentful/forma-36-react-components';
+
+const [isDropdownOpen, setDropdownOpen] = React.useState(false)
+
+<Dropdown
+  isOpen={isDropdownOpen}
+  onClose={() => setDropdownOpen(false)}
+  toggleElement={
+    <IconButton
+      aria-label="Open Actions"
+      icon={<MoreVerticalIcon size="small" />}
+      onClick={(event) => {
+        setDropdownOpen((isOpen) => !isOpen)
+      }}
+    />
+  }>
+  <DropdownList>
+    <DropdownListItem>
+      First action
+    </DropdownListItem>
+    <Dropdown submenuToggleLabel="Open submenu">
+      <DropdownList>
+        <DropdownListItem>
+          Submenu action 1
+        </DropdownListItem>
+        <DropdownListItem>
+          Submenu action 2
+        </DropdownListItem>
+      </DropdownList>
+    </Dropdown>
+    <DropdownListItem>
+      Second action
+    </DropdownListItem>
+  </DropdownList>
+</Dropdown>
+```
+
+Forma v4:
+
+```tsx static=true
+import { Menu, IconButton } from '@contentful/f36-components';
+
+<Menu>
+  <Menu.Trigger>
+    <IconButton
+      aria-label="Open Actions"
+      icon={<MoreVerticalIcon size="small" />}
+    />
+  </Menu.Trigger>
+  <Menu.List>
+    <Menu.Item onClick={() => {/* do some action */}}>First action</Menu.Item>
+    <Menu.Submenu>
+      <Menu.SubmenuTrigger>Open submenu</Menu.SubmenuTrigger>
+      <Menu.List>
+        <Menu.Item>Submenu action 1</Menu.Item>
+        <Menu.Item>Submenu action 2</Menu.Item>
+      </Menu.List>
+    </Menu.Submenu>
+    <Menu.Item onClick={() => {/* do some action */}}>Second action</Menu.Item>
+  </Menu.List>
+</Menu>
+```
+
+#### `Dropdown` to `Popover`
+
+**Popover case with checkboxes:**
+
+Importent things to take into account in the following example:
+- Unlike `Menu`, `Popover` is a controlled component. So you have to pass callbacks and state in order to make it work.
+- We used `FocusLock` from [react-focus-lock](https://github.com/theKashey/react-focus-lock) to enhance a11y. So user can't leave the popover with "Tab" navigation.
```

Comment:
```suggestion
- We used `FocusLock` from [react-focus-lock](https://github.com/theKashey/react-focus-lock) to enhance a11y. It traps users' focus in the popover context, user will not be able to leave the component when using "Tab" navigation. 
```

Reply:
I wanted to explain a bit more how it works, but I don't know if I did a good job here 😬